apps/web/components/HomePage/ArrayDoubleColumn.tsx

```diff
@@ -1,14 +1,43 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import styles from "styles/leaderBoard.module.scss";
 import textStyle from "styles/text.module.scss";
 import entityStyles from "styles/entity.module.scss";
+import LeaderboardEntity from "./LeaderboardEntity";
+import User from "../../interface/UserInterface";
 
 export default function ArrayDoubleColumn(props: {
   title: string;
-  list: JSX.Element[];
+  handleClick: (e: {
+    index: number;
+    openMenu: boolean;
+    setOpenMenu: React.Dispatch<React.SetStateAction<boolean>>;
+  }) => void;
 }): JSX.Element {
   const [columnNum, setColumnNum] = useState(1);
   const [pageNum, setPageNum] = useState(1);
+  const [leaderBoardList, setLeaderBoardList] = useState([<></>]);
+  const [column1, setColumn1] = useState([<></>]);
+  const [column2, setColumn2] = useState([<></>]);
+
+  useEffect(() => {
+    const tmp: JSX.Element[] = [];
+    fetch(`${process.env.NEXT_PUBLIC_BACKEND_BASE_URL}/leaderBoard`)
```

Comment:
The backend address has changed as you will see when your merge or rebase from main. It is now prefixed with "/api/" instead of "${process.env.NEXT_PUBLIC_BACKEND_BASE_URL}/"
Please stick with relative url for flexibility reasons (domain name should be autoscoped by the browser as we do not have a registered domain name)

Reply:
i don't understand ... if i replace "${process.env.NEXT_PUBLIC_BACKEND_BASE_URL}/" with "/api" it doesn't work. 
"The backend address has changed as you will see" ... i didn't see ahah sorry

Reply:
done